bom/application/pom.xml

```diff
@@ -3302,6 +3302,12 @@
                 <groupId>io.smallrye</groupId>
                 <artifactId>smallrye-health</artifactId>
                 <version>${smallrye-health.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>jakarta.json</groupId>
+                        <artifactId>jakarta.json-api</artifactId>
```

Comment:
Question, do you need JSON-P around or is it just optional?

Reply:
yes, we also now have a non-CDI plain option of using sr-health so I needed to ship this too, unfortunately.

Reply:
Ah OK, I just checked and the dependency is already there. Fine then.